drivers/scheduler/k8s/k8s.go

```diff
@@ -4100,6 +4118,40 @@ func (k *K8s) ValidateAutopilotRuleObjects() error {
 	return nil
 }
 
+// GetIOBandwidth takes in the pod name and namespace and returns the IOPs speed
+func (k *K8s) GetIOBandwidth(podName string, namespace string) (int, error) {
+	logrus.Infof("Getting the IO Speed in pod %s", podName)
+	pod, err := k8sCore.GetPodByName(podName, namespace)
+	if err != nil {
+		return 0, fmt.Errorf("error in getting FIO PODS")
+	}
+	logOptions := corev1.PodLogOptions{
+		TailLines: getInt64Address(15),
+	}
+	log, err := k8sCore.GetPodLog(pod.Name, pod.Namespace, &logOptions)
+	if err != nil {
+		return 0, err
+	}
+	outputLines := strings.Split(log, "\n")
+	for _, line := range outputLines {
+		if strings.Contains(line, "bw=") {
+			re := regexp.MustCompile(`[0-9]+KB/s`)
```

Comment:
@aghodke1312 it can be B/s or KB/s MB/S

Reply:
No from the test and discussion with Weiwei it looks like it will be KB/s